VMBackup/main/backuplogger.py

```diff
@@ -46,8 +50,9 @@ def log(self, msg, local=False, level='Info'):
             else:
                 log_msg = "{0}  {1}  {2} \n".format(str(datetime.datetime.now()) , level , msg)
                 self.log_to_con(log_msg)
-            if self.enforced_local_flag_value != None:
-                local = self.enforced_local_flag_value
+            local = True
+            if self.enforced_local_flag_value == False:
+                local = False
             if(local):
```

Comment:
if this variable doesn't serve any purpose, we can directly use enforced_local_flag_value  here.
I assume 'local' variable is not removed as to not break somewhere

Reply:
Done the change to directly use enforced_local_flag_value.
Yes, local is not removed to not break.